src/site/content/en/blog/native-lazy-loading/index.md

```diff
@@ -0,0 +1,305 @@
+
+```html
+<img src="…" alt="…" loading="lazy" intrinsicsize="250x200" width="450">
+<!-- lazy loaded -->
+```
+
+<div class="w-aside w-aside--note">
+  Take a look at this <a href="https://mathiasbynens.be/demo/img-loading-lazy">demo</a> to see how the <code>loading</code> attribute works with 100 pictures.
+</div>
+
+### iframe loading
+
+The `loading` attribute affects iframes differently than images, depending on whether the iframe is
+hidden. (Hidden iframes are often used for analytics or communication purposes.) Chrome uses the
+following criteria to determine whether an iframe is hidden:
+
+- The iframe's width and height are 4 px or smaller.
+- `display: none` or `visibility: hidden` is applied.
+- The iframe is placed off-screen using negative X or Y positioning.
+
+If an iframe meets any of these conditions, Chrome considers it hidden and won't lazy load it in
+most cases. Iframes that _aren't_ hidden will only load when they're within a certain distance from
+the viewport. A similar placeholder also shows for lazily loaded iframes that are still being
+fetched.
+
+### Feature Policy
+
+Developers can override the default behavior through [feature
+policies](https://developers.google.com/web/updates/2018/06/feature-policy):
+
+- `loading-image-default-eager`: changes the default behavior of the `loading` attribute for images.
+- `loading-frame-default-eager`: changes the default behavior of the `loading` attribute for iframes.
+
+The Feature-Policy HTTP header can be used to control both features. For example, the following
+header can be used for all images on the page:
+
+```
+Feature-Policy: loading-image-default-eager 'none'
+```
+
+This lazily loads every image below the device viewport unless `loading=eager` is used.
+
+```html
+<img src="kitten.jpg" alt="…">
+<!-- loaded lazily -->
+<img loading="eager" src="puppy.jpg" alt="…">
+<!-- loaded eagerly -->
+```
+
+The same can be done for iframes:
+
+```
+Feature-Policy: loading-frame-default-eager 'none'
+```
+
+To learn more about how both these policies work, as well as how to allow the feature for only
+certain origins on a page, take a look at the policy proposals for both
+[images](https://github.com/w3c/webappsec-feature-policy/blob/master/policies/loading-image-default-eager.md)
+and
+[frames](https://github.com/w3c/webappsec-feature-policy/blob/master/policies/loading-frame-default-eager.md).
+
+<div class="w-aside w-aside--note">
+  Complete functionality for both feature policies is expected to launch by Chrome 77.
+</div>
+
+## FAQ
+
+### Can I change how close an image or iframe needs to be before a load is triggered?
+
+These values are hardcoded and can't be changed through the API. However, they may change in the
+future as the Chrome team experiments with different threshold distances and variables.
+
+### Can CSS background images take advantage of the `loading` attribute?
+
+No, it can only be used with `<img>` tags.
```

Comment:
"it can _currently_ only be used with `<img>` tags." per https://github.com/w3c/webappsec-feature-policy/issues/311?